packages/verifiable-credentials/src/types/verifiableCredential.ts

```diff
@@ -0,0 +1,17 @@
+import { EVMNFT } from "@crossmint/client-sdk-base/src/types/blockchain";
```

Comment:
shouldnt import like this

instead add `@crossmint/client-sdk-base` as a dependency in the vc package.json, then import only from `@crossmint/client-sdk-base`

Reply:
and actually, because you are only using the type, you can add it as a **dev dependency** rather than a dependency